app/main.py

```
@@ -1,15 +1,72 @@
+        "http://127.0.0.1:3000",
+    ],
+    allow_credentials=True,
+)
+
+app.include_router(api_router, prefix="/v1")
+
+
+# --- восстановление active_datasets при старте ---
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
+from datetime import datetime, timezone
```

Ruff (0.14.8)

Redefinition of unused `datetime` from line 10

Remove definition: `datetime`

(F811)